server/services/websockets.ts

```diff
@@ -49,8 +49,8 @@ export default function init(app: Koa, server: http.Server) {
 
   io.adapter(
     socketRedisAdapter({
-      pubClient: client,
-      subClient: subscriber,
+      pubClient: Redis.Client,
+      subClient: Redis.Subscriber,
```

Comment:
What made you do this stylistic change – could we not have kept the original exports or is there something I'm missing?

The capitalization makes it look like these are classes that need initializing.

Reply:
No, you are absolutely right, this not a necessary change. I just think it is good practice to store global singletons of a class as static properties of said class, because is makes the relationship between them very clear everywhere you import/use them. In fact, many projects I am working on use this pattern, e.g. Apple even recommends this pattern in their developer guidelines.

You are right about the naming though. Maybe we can rename the clients to `Redis.defaultClient`/`Redis.defaultSubscriber` or `sharedClient`. Would that work for you? Otherwise I'll revert those changes.

Reply:
> Maybe we can rename the clients to Redis.defaultClient/Redis.defaultSubscriber 

This works for me